packages/typespec-test/test/openai_generic/generated/typespec-ts/review/openai-generic.api.md

```diff
@@ -185,120 +185,120 @@ export interface CreateCompletionResponse {
     usage?: CompletionUsage;
 }
 
-// @public (undocumented)
+// @public
 export interface CreateEditRequest {
     input?: string | null;
     instruction: string;
-    model: string | "text-davinci-edit-001" | "code-davinci-edit-001";
+    model: "text-davinci-edit-001" | "code-davinci-edit-001";
```

Comment:
seems `string` type is missing, is this expected?

Reply:
I think so, the model property type is extensible enum, and according to https://github.com/Azure/autorest.typescript/issues/2420#issuecomment-2166703162 fixed union is our option if no experimentalExtensibleEnums is enabled.